go/vt/vtadmin/api_test.go

```diff
@@ -41,10 +43,13 @@ import (
 	"vitess.io/vitess/go/vt/vtctl/vtctldclient"
 	"vitess.io/vitess/go/vt/vttablet/tmclient"
 
+	querypb "vitess.io/vitess/go/vt/proto/query"
+	"vitess.io/vitess/go/vt/proto/tabletmanagerdata"
 	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
 	vtadminpb "vitess.io/vitess/go/vt/proto/vtadmin"
 	vtctldatapb "vitess.io/vitess/go/vt/proto/vtctldata"
 	"vitess.io/vitess/go/vt/proto/vttime"
+	vtadmintestutil "vitess.io/vitess/go/vt/vtadmin/testutil"
```

Comment:
sadly vscode doesn't always do this right, but for consistency this should be up with the non-proto vitess imports

Reply:
✅